### ISSUE TITLE
Support for Proxmox 7.4 (after release of "official" dark mode) - few issues with pvediscord dark

### ISSUE THREAD

**Issue by accessiblepixel** [Weilbyte/PVEDiscordDark#160]
Greetings and salutations! :)

Proxmox seems to have added their own official darkmode now, but I'm not really a fan. I've gotten very used to how things look with PVEDiscordDark so I went to reinstall it to re-enable it, and it mostly works.

There are a few icons that don't seem to be changed (the node icon, and the storage icon - they're quite bright/white) and the 'proxmox' logo looks a bit weird (orange and black on grey).

The graphs now use the black from the official dark mode skin (when you enable 'dark' in the options) so it doesn't look too out of place, but it'd be nicer if it was the same grey it used to be, but if it's not something that's easy to change I can live with that.

Of course the installer says that it's not a supported version (which makes sense!), but I do hope that it can be fully supported again soon.

I'd like to say firstly thank you for making this project available, I've used it for quite some time (and my eyes thank you!) and really appreciate the work you've put into it so far.

Looking forward to your reply.

Kind regards,
Jessica

**Comment by djraw** [comment]
One important thing to do with the new 7.4 pve, though it seems paradox - after applying this Discord theme you should set the webinterface to "light".

Most changes the script applies only change the former standard, the light theme.

**Comment by accessiblepixel** [comment]
Thanks for the reply - I swapped back to light mode, uninstalled the theme, reinstalled and the icons and logo now looks as it should with PVEDiscordDark however my graphs are now white and the number/details on the axis are unreadable. 
I'm guessing I'm doing something wrong?

**Comment by djraw** [comment]
> I'm guessing I'm doing something wrong? 

No, you are doing everything right. Unfortunately the changes to the graphs don't work anymore with 7.4

So either you have light graphs but most icons etc. work, or dark graphs but many other issues.

Anyway, as the script states for now 7.4 is unsupported and this time it really is. ;)

Honestly I use the included dark mode for now and hope for an update to this script.. or for proxmox to include it as a selectable option.

**Comment by accessiblepixel** [comment]
Totally! I know it's an unsupported option, but I personally feel for open-source software and what not the price of admission is filing issues when I can so that people know that there's an issue and that people are using whatever it is :)

I think the workaround for me would be having the dark mode with the slightly odd icons, since the graphs not being usable is a worse trade off just for some less bright icons, and I look forward to if/when there's a new fix for us that love this theme (I do like that they've included a darkmode by default now (it's a about time! 😋) but I don't like change and I really like the way PVEDiscordDark looks.

I'm willing to throw in a couple of dollars to aid development of something that I basically use every day if that will help 😃  (but don't see an obvious way listed in the repo and not sure if the author(s) would like donations anyway).

Kind regards,
Jessica

**Comment by sterzy** [comment]
just to throw in my two cents, at least for charts adapting this theme would be fairly easy, you can simply set a couple [css variables now](https://git.proxmox.com/?p=proxmox-widget-toolkit.git;a=blob;f=src/proxmox-dark/scss/other/_charts.scss;h=5c67282896fb5295fa978e4cda511f6ff8e100ff;hb=HEAD), and the chart and gauge components will take care of the rest. this even works with a `prefers-color-scheme` css media query, as the components [now listen on changes here](https://git.proxmox.com/?p=proxmox-widget-toolkit.git;a=blob;f=src/panel/RRDChart.js;h=dc5940c6566e236bf85d2df711267539bd2ecf61;hb=HEAD#l186).

this and some other changes should obsolete the [`PVEDiscordDark.js`](https://github.com/Weilbyte/PVEDiscordDark/blob/master/PVEDiscordDark/js/PVEDiscordDark.js) afaict.

**Comment by Weilbyte** [comment]
Really glad to hear how much you like the theme; as for donations, I am not accepting any at the moment and do not plan to. I still have yet to take a proper look at the new version, but it does seem promising that the charts now get their colors from CSS variables. I am not sure how the project would continue with future versions though since the inclusion of a built-in dark mode, as any further changes would require both compatibility with the current version and also the previous ones (but for those I am thinking the releases would be fine as no further changes are needed); hopefully we can figure something out.

**Comment by r4yzs** [comment]
Hello,

This new dark theme is very ugly. 

I hope a new version of this discord theme...

**Comment by AntonUden** [comment]
setting the proxmox theme to light fixes the icons but makes the graphs have a white backgrounds instead
